templates/ocs-deployment/CephBlockPool.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: {{ cephblockpool_name | default('ocsci-cephblockpool') }}
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+spec:
+  failureDomain: {{ failureDomain | default('host') }}
+  replicated:
+    size: {{ size | default('2') }}
```

Comment:
Let us have the default replica size as '3' instead of '2' as that is what will be supported.

Reply:
Done